net-tools/netwatch/src/interfaces.rs

```diff
@@ -341,7 +339,7 @@ pub async fn default_route_interface() -> Option<String> {
 /// Likely IPs of the residentla router, and the ip address of the current
 /// machine using it.
 #[derive(Debug, Clone)]
-pub(crate) struct HomeRouter {
+pub struct HomeRouter {
```

Comment:
I wonder if we should look into upstreaming this `HomeRouter` into `netdev` longterm